src/overlays/actors/ovl_Nameplate/ac_nameplate.c

```diff
@@ -38,7 +38,7 @@ void aNP_actor_dt(Actor* thisx UNUSED, Game_Play* game_play UNUSED) {
 
 void aNP_set_talk_info(Actor* thisx) {
     Color_RGBA8 color;
-    char villagerName[8];
+    u8 villagerName[8];
```

Comment:
This is a string, isn't it? Why use a `u8` array instead of a `char` array?

Reply:
As these are with the custom in-game encoding, would these still be considered String/char? That's why I was unsure if to change them or not